src/System.Net.Sockets/tests/FunctionalTests/TimeoutTest.cs

```diff
@@ -58,25 +62,33 @@ public void ReceiveTimesOut_Throws(bool forceNonBlocking)
                 remoteSocket.Connect(IPAddress.IPv6Loopback, port);
 
                 Socket acceptedSocket = localSocket.EndAccept(localAsync);
-                acceptedSocket.ReceiveTimeout = TestSettings.FailingTestTimeout;
+                acceptedSocket.ReceiveTimeout = Timeout;
 
                 acceptedSocket.ForceNonBlocking(forceNonBlocking);
 
+                DateTime start = default(DateTime);
+
                 SocketException sockEx = Assert.Throws<SocketException>(() =>
                 {
+                    start = DateTime.Now;
```

Comment:
Use `DateTime.UtcNow` to avoid timezones and daylight savings? Although, I guess test failures due to this would be pretty rare.